src/frontend/src/lib/components/receive/ReceiveAddressModal.svelte

```diff
@@ -24,42 +25,46 @@
 	let currentStep: WizardStep | undefined;
 	let modal: WizardModal;
 
-	let qrCodeAddress: undefined | string;
-	let qrCodeAddressLabel: undefined | string;
-	let qrCodeAddressToken: Token | undefined;
+	let address: undefined | string;
+	let addressLabel: undefined | string;
+	let addressToken: Token | undefined;
+	let copyAriaLabel: string | undefined;
 
-	const displayQRCode = ({
-		detail: { address, addressLabel, addressToken }
-	}: CustomEvent<ReceiveQRCode>) => {
-		qrCodeAddress = address;
-		qrCodeAddressLabel = addressLabel;
-		qrCodeAddressToken = addressToken;
+	const displayQRCode = ({ detail }: CustomEvent<ReceiveQRCode>) => {
```

Comment:
It was original destructed but now, the local state variables have the same name as the assignated values.

```
address = detail.address;
```

So I can destruct but I would have to come up with names but had no idea. For a while I had something like

```
const displayQRCode = ({ detail: {a} }:

address = a;
```

because following cannot work

```
address = address
```

But ultimately found those pseudo names not readable. Hence, not destructing.

Makes sense?

Reply:
ah understood now! yes, makes sense